src/aria/utils/Orientation.js

```diff
@@ -33,6 +37,8 @@ module.exports = Aria.classDefinition({
      * Adding a listener while initializing Orientation to listen to the native orientationchange event.
      */
     $constructor : function () {
+        this.$logWarn("This class is deprecated and will be removed from the framework, please use aria.utils.Device instead");
+
```

Comment:
Also I'd change `This class is deprecated` to `aria.utils.Orientation class is deprecated`